e2e/tests/patient/patient-maintenance.e2e-spec.ts

```diff
@@ -0,0 +1,119 @@
+        allergyDetailPage.getSignsInput().setText(allergyData[1]);
+        allergyDetailPage.getSymptomsInput().setText(allergyData[2]);
+        allergyDetailPage.getButtonSubmit().click();
+
+        TestUtil.checkWidgetPresentAndDisplayed(mainPage.getConfigTabs(), 'Config. tabs');
+        TestUtil.checkWidgetPresentAndDisplayed(mainPage.getAllergyAddButton(), 'Allergy Add button');
+        TestUtil.checkWidgetPresentAndDisplayed(mainPage.getAllergyRefreshButton(), 'Allergy Refresh button');
+        TestUtil.checkWidgetPresentAndDisplayed(mainPage.getAllergyOptionsButton(), 'Allergy Options button');
+        TestUtil.checkWidgetPresentAndDisplayed(mainPage.getAllergyGrid(), 'Allergy grid');
+
+        TestUtil.checkNumber(mainPage.getAllergyGrid().getNumberOfRows(), 'Number of Allergies', 1);
+    }
+
+    function createPatientItem(patientData: string[]) {
+        patientMaintenancePage.getButtonAdd().click();
+        NavigationService.checkPageIsPresentAndDisplayed(patientDetailPage);
+        patientDetailPage.getNameInput().setText(patientData[0]);
+        patientDetailPage.getSurnameInput().setText(patientData[1]);
+        patientDetailPage.getEmailInput().setText(patientData[2]);
+        patientDetailPage.getAddressStreetInput().setText(patientData[3]);
+        patientDetailPage.getAddressCityInput().setText(patientData[4]);
+        patientDetailPage.getAddressZipInput().setText(patientData[5]);
+        patientDetailPage.getAddressCoordinatesInput().setText(patientData[6])
+        patientDetailPage.getButtonSubmit().click();
+        NavigationService.checkPageIsPresentAndDisplayed(patientMaintenancePage);
+        TestUtil.checkNumber(patientMaintenancePage.getPatientsGrid().getNumberOfRows(), 'Number of Patients', 1);
+    }
+
+    function assignAllegryToPatient(allergyNum: number) {
+        patientDetailPage.getAddButton().click();
+        NavigationService.checkPageIsPresentAndDisplayed(patientAllergyDetailPage);
+        patientAllergyDetailPage.getAllergyCombobox().click();
+        patientAllergyDetailPage.getAllergyList().get(0).click();
+
+/*
+        patientAllergyDetailPage.getAssertedDate().setValue('01/01/2019');
+        patientAllergyDetailPage.getLastOccurrenceDate().setValue('02/02/2019');
+*/
+        patientAllergyDetailPage.getAllergyNotes().setText('aslhslkjhdsalkjah');
+        patientAllergyDetailPage.getSubmitButton().click();
+        TestUtil.checkNumber(patientDetailPage.getAllergyGrid().getNumberOfRows(), 'Number of Allergies', 1);
+    }
+
+    it('Access to the Patient Management Dialog', () => {
+        NavigationService.checkPageTitleAndButtons(patientMaintenancePage,patientMaintenancePage.title, patientMaintenancePage.buttons);
+        const titles = ['', 'Name', 'Surname', 'Email'];
+        expect(patientMaintenancePage.getPatientsGrid().getGridHeader()).toEqual(titles);
+    });
+
+    it('Assign allergy to a patient', () => {
```

Comment:
This step has some expected results twice, we need to review it in order to have all the asserts but just one formally documented.

![image](https://user-images.githubusercontent.com/1970034/69875164-f44c1580-12bc-11ea-9f6d-a53a123ec4df.png)